packages/driver/src/multi-domain/communicator.ts

```diff
@@ -169,14 +169,14 @@ export class SpecBridgeDomainCommunicator extends EventEmitter {
    * @param {Cypress.ObjectLike} data - any meta data to be sent with the event.
    */
   toPrimary (event: string, data?: Cypress.ObjectLike, options: { syncGlobals: boolean } = { syncGlobals: false }) {
-    debug('<= to Primary ', event, data, document.domain)
+    debug('<= to Primary ', event, data, location.origin)
```

Comment:
```suggestion
    debug('<= to Primary ', event, data, location.originPolicy)
```

Reply:
location.origin === location.originPolicy in this instance because the location is from the spec bridge, but to reduce confusion I'll grab the origin policy anyway.